File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/ContextMenuPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.GroupBox.Features
{
    public partial class ContextMenuPage : UserControl
    {
        public ContextMenuPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handels Click event for Context menu item.
        /// Shows message box.
        /// </summary>
        private void menuItem_Click(object sender, EventArgs e)
        {
            MenuItem menuItem = sender as MenuItem;
            if (menuItem != null)
            {
                this.mobjLabel.Text = string.Format("You've selected '{0}' menu item.", menuItem.Text);
            }
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/GroupBox/Features/ContextMenuPage.vb
Namespace CompanionKit.Controls.GroupBox.Features

	Public Class ContextMenuPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handels Click event for Context menu item.
        ''' Shows message box.
        ''' </summary>
        ''' <param name="sender"></param>
        ''' <param name="e"></param>
        ''' <remarks></remarks>
        Private Sub MenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles mobjMenuItem1.Click, mobjMenuItem2.Click, mobjMenuItem3.Click, mobjMenuItem4.Click, mobjMenuItem5.Click
            Dim menuItem As MenuItem = TryCast(sender, MenuItem)
            If (Not menuItem Is Nothing) Then
                Me.mobjLabel.Text = String.Format("You've selected '{0}' menu item.", menuItem.Text)
            End If

        End Sub
	End Class

End Namespace